tic_tac_toe/game.py

```diff
@@ -87,18 +147,60 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    #create board object
+    board = game['board']
+    #check illegal moves
+    if game['winner'] or _board_is_full(board):
+        raise InvalidMovement('Game is over.')
+    if player != game['next_turn']:
+        raise InvalidMovement('"{}" moves next.'.format(game['next_turn']))
+    if not _position_is_valid(position):
+        raise InvalidMovement('Position out of range.')
+    if not _position_is_empty_in_board(position, board):
+        raise InvalidMovement('Position already taken.')
+    #make actual move
+    board[position[0]][position[1]] = player
+    #set winner
+    winner = _check_winning_combinations(board, player)
+    #explore options if a winner exists
```

Comment:
You don't necessarily need to set `winner` only to set `game['winner']` to it's value a couple of lines later. You could simply use `game['winner'] = _check_winning_combinations(board, player)` This works because `_check_winning_combinations` is supposed to return `None` (if a `return` isn't specifically triggered a function returns `None`) if there is no winner.